webapp/src/main/java/life/qbic/views/login/passwordreset/PasswordResetHandler.java

```diff
@@ -51,7 +66,6 @@ public void onPasswordResetSucceeded() {
 
     @Override
     public void onPasswordResetFailed() {
-        //todo
-        throw new NotImplementedException();
+        showPasswordResetFailedError();
```

Comment:
If we handle use case exceptions, i think we should do it with max satisfaction as possible to the user.

"The Password Reset has failed", "Please try again."

The user does not know, what they have done wrong. So i think we should increase transparency a little bit, and introduce methods in the output interface.

For example we can invoke a method `onUnknownEmailAddress()` if the email does not belong to any registered and active user account. We can further introduce a `onUnexpectedException()`, that will redirect the user to support@qbic.zendesk.com to handle more severe scenarios.